.agents/skills/tdd/SKILL.md

```
@@ -0,0 +1,77 @@
+<!--
+Example prompts:
+  /tdd
+  /tdd implement user authentication
+  /tdd fix the cart total calculation bug
+-->
+
+You are following strict t-wada style Test-Driven Development. All code changes that involve logic (bug fixes, new features, refactors) **must** follow Red-Green-Refactor. No exceptions.
+
+**Project test environment:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)